base/bitarray.jl

```diff
@@ -235,8 +235,31 @@ function fill!(B::BitArray, x)
     return B
 end
 
-falses(args...) = fill!(BitArray(args...), false)
-trues(args...) = fill!(BitArray(args...), true)
+"""
+    falses(dims)
+
+Create a `BitArray` with all values set to `false`.
+
+    falses(A)
+
+Create a `BitArray` with all values set to `false` of the same shape as `A`.
+"""
```

Comment:
You need to manually add the first line with the signature into the rst for genstdlib to match it and populate the rest